src/components/JoinGroupForm/JoinGroupForm.tsx

```diff
@@ -1,47 +1,45 @@
-import React, { useState, useRef } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { RootState } from 'store/store';
+import { useSelector, useDispatch } from 'react-redux';
 
 import { InputField } from 'components/InputField/InputField';
 import { Button } from 'components/Button/Button';
 import Modal from 'components/Modal/Modal';
 
+import { Class } from 'types/Class/Class';
+import { getClass, joinClass } from 'utils/db/db.utils';
+
+import { updateToast, GENERAL_ERRORS } from 'store/toast/toastSlice';
+import { setLoading, removeLoading } from 'store/loading/loadingSlice';
+
 import { inputRules } from 'utils/inputRules/groupRules';
 
 import { correctState } from 'utils/inputRules/generalRules';
 import styles from './JoinGroupForm.module.css';
 
-type ClassInfo = {
-  class_id: string;
-  class_name: string;
-  teacher: string;
-  days: string;
-  time: string;
-};
-
-const dummyInfo = <T,>(): Promise<T> =>
-  new Promise((resolve) => {
-    setTimeout(() => {
-      resolve({
-        class_id: 'ITC023',
-        class_name: 'Tecnologias II',
-        teacher: 'Claudia Perez',
-        days: 'LU-JU',
-        time: '09:00-11:00',
-      } as T);
-    }, 1000);
-  });
-
 export default function JoinGroupForm() {
   const [isSubmitDisabled, setIsSubmitDisabled] = useState(true);
```

Comment:
I couldn't find where you use setIsSubmitDisabled. If I understand correctly, this state is used to let the form know which is the last focusable element, but I am not sure where that value is changed.

Reply:
This was not a change on this commit. I found it on line 21 in case GitHub don't display it since it is not a change

Reply:
Not exactly, the setter also handles if the submit button is disabled or not (based on input checking)